src/list/manage/components/item-details.css

```diff
@@ -7,15 +7,46 @@
   background-color: #f9f9fa;
 }
 
-.item-details > h1 {
+.item-details > header {
+  position: relative;
+}
+
+.item-details > h1,
```

Comment:
will this conflict with identical work in #87 ?

Reply:
Hmm, it probably will, but we should probably also just rebase / revise against whichever lands first? Not sure how to pre-emptively handle that well